test/assignment1/username-from-conf-file.h

```diff
@@ -1,34 +1,45 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <ctype.h>
 
 /**
  * @return the name of the user from the conf file, or
  * an empty string if not found.  Must be freed by the caller
  */
 static inline char *malloc_username_from_conf_file()
 {
-    size_t len = 0;
-    char *buffer = malloc(1);
+    static const uint8_t BASE_LEN = 16;
+    size_t len = BASE_LEN;
+    char *buffer = malloc(BASE_LEN);
```

Comment:
malloc(1) and passing len 0 guarantees there will have to be a realloc in getline(); might as well start with a reasonable guess that will probably succeed without realloc

Reply:
This relates to my review comment, I didn't see this when writing mine.  This is mostly just a matter of preference I guess, but I think it should be more clearly documented what's happening.